packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -3346,14 +3344,6 @@ const getSSHTunnelFunctions = (): SSHTunnelFunctions => ({
 		await Container.get(SSHClientsManager).getClient(credentials),
 });
 
-const getSchedulingFunctions = (workflow: Workflow): SchedulingFunctions => {
```

Comment:
This was not used?

Reply:
this was only used in poll and trigger contexts. with those two extracted, this function isn't needed here. This code now lives in `helpers/scheduling-helpers` now.